proto/plan.proto

```diff
@@ -551,6 +552,7 @@ message InsertValues {
 	repeated ColDef other_cols = 4;
 	repeated string order_attrs = 5;
 	repeated column columns = 6;
+	ColDef compositePKey = 7;
```

Comment:
use:  composite_pkey will be better in proto file

Reply:
ok, I will change it